packaging/installer/methods/freebsd.md

```diff
@@ -19,7 +26,7 @@ pkg install bash e2fsprogs-libuuid git curl autoconf automake pkgconf pidof Judy
 Download Netdata:
 
 ```sh
-fetch https://github.com/netdata/netdata/releases/download/v1.25.0/netdata-v1.25.0.tar.gz
+fetch https://github.com/netdata/netdata/releases/download/v1.26.0/netdata-v1.26.0.tar.gz
```

Comment:
Could mention that latest can be checked via:

- browser: https://github.com/netdata/netdata/releases/latest
- cmd: `basename $(curl -Ls -o /dev/null -w %{url_effective} https://github.com/netdata/netdata/releases/latest)`

I am not sure `curl` is installed on pfsense

```cmd
0 ~ $ basename $(curl -Ls -o /dev/null -w %{url_effective} https://github.com/netdata/netdata/releases/latest)
v1.26.0
```

Reply:
`curl` is not on pfSense as far as my research goes, maybe for FreeBSD.

Reply:
Anyway it is a way to get the latest - via browser or cmd (not directly from the pfsense box but from your laptop/other server)

Reply:
Oh, good call!